docs/user-guide.md

```diff
@@ -1,35 +1,52 @@
 # User Guide
 
-## Advanced
-
-### Building the base and node images
-#### Base image
-`kind` runs a local kubernetes cluster by using Docker containers as "nodes."
-`kind` uses the [`node-image`][node image] to run kubernetes artifacts, such 
-as `kubeadm` or `kubelet`.
-The `node-image` in turn is built off the [`base-image`][base image], which
-installs all the dependencies needed for Docker and Kubernetes to run in a
-container.
+## Creating a Cluster
 
-To build the `base-image` we use the `build` command:
+Creating a kubernetes cluster is as simple as:
 ```
-$ kind build base-image
+$ kind create cluster
 ```
 
-If you want to specify the path to the base image source files you can use the
-`--source` flag.
-If `--source` is not specified, `kind` has enbedded the contents of the in
-default base image in [`pkg/build/base/sources`][pkg/build/base/sources] and
-will use this to build it.
+This will bootstrap a kubernetes cluster using a pre-built 
+[node image][node image] - you can find it on docker hub
+[`kindest/node`][kindest/node]. 
+If you desire to build the node image yourself see the 
+[building image](#building-images) section.
+To specify another image use the `--image` flag.
 
-By default, the base image will be tagged as `kindest/base:latest`.
-If you want to change this, you can use the `--image` flag.
+By default, the cluster will be given the name `kind-1`. "1," here, is the
```

Comment:
reminder to self: I think _maybe_ we should stop auto-prefixing this and just use context name directly cc @munnerz 🤔